android/src/main/java/com/rudderstack/android/RudderAnalytics.kt

```diff
@@ -92,14 +93,10 @@ fun createInstance(
             storage,
             initializationListener
         ).also { analyticsInstance ->
-            AnalyticsRegistry.register(writeKey, analyticsInstance)
+            AnalyticsRegistry.register(DEFAULT_INSTANCE, analyticsInstance)
         }
 }
 
-fun getInstance(writeKey: String): Analytics? {
```

Comment:
make this method private and call the same from line 87

Reply:
This method is not needed at all. Is actually the same method, with the same parameters, but only private for no particular reason. The public `fun getInstance()` in actually enough.